### PR TITLE
feat: Adding/Removing item to/from lists in bulk endpoint

### LINKED ISSUE #120: Create the pick & unpick in bulk endpoint
In order to pick and unpick items from different lists a the same time, we need to a add a new endpoint.
This endpoint will have the following path and method: `POST /v1/picks/:itemId` and will receive a list of list ids in which the item will be picked or unpicked.

This is an example of the request body:
```json
{
  "unpickedFrom": [],
  "pickedFor": []
}
```

On which `unpickedFrom` is an array of list ids where to unpick the item from and the `pickedFor` is an array of list ids where to add that item into.

### PR DESCRIPTION
**Pull request by kevinszuchet**
Closes #120

### CODE REVIEW

File: src/ports/picks/component.ts [hunk 1]
```diff
@@ -68,5 +73,43 @@ export function createPicksComponent(components: Pick<AppComponents, 'pg'>): IPi
...
+    const allLists = [...pickedFor, ...unpickedFrom].join(', ')
+    const { rows } = await pg.query<{ total: number }>(
+      SQL`SELECT COUNT(1) total FROM favorites.lists
+      LEFT JOIN favorites.acl ON favorites.lists.id = favorites.acl.list_id
+      WHERE favorites.lists.id IN (${allLists}) AND favorites.lists.user_address != ${userAddress}
+      AND (favorites.acl.permission != ${Permission.EDIT} OR favorites.acl.grantee NOT IN (${userAddress}, ${GRANTED_TO_ALL}))`
+    )
+
+    if (rows[0]?.total > 0) {
+      throw new ForbiddenLists()
```
Comment (LautaroPetaccio) [inline src/ports/picks/component.ts:91]: We could return a "ListsNotFound" to be consistente with the other status we have, what do you think?

File: test/unit/picks-handler.spec.ts [hunk 2]
```diff
@@ -459,3 +464,143 @@ describe('when getting the picks for an item', () => {
...
+
+  describe('and the process fails with a lists not found error', () => {
+    const pickedFor = ['list-id', 'another-list-id']
+    const unpickedFrom = ['a-different-list-id']
+    beforeEach(() => {
+      jsonMock.mockResolvedValueOnce({ pickedFor, unpickedFrom })
+      pickAndUnpickInBulkMock.mockRejectedValueOnce(new ListsNotFoundError([...pickedFor, ...unpickedFrom]))
+    })
+
+    it('should return a response with a message saying that there are some lists inaccessible to the user and the 403 status code', () => {
```
Comment (LautaroPetaccio) [inline test/unit/picks-handler.spec.ts:545]:
```suggestion
    it('should return a response with a message saying that there are some lists inaccessible to the user and the 404 status code', () => {
```

**Review by LautaroPetaccio — APPROVED** [review]
Looks incredible! 👍  I've left two small typos and something that might be improvable. If you find the comments useful, change the code and merge it.

File: src/ports/vp/queries.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  // If the snapshot query fails, the power will be undefined
+  // Try to set the VP to 0 without overwriting it if it already exists
+  if (!power) {
+    query.append(SQL`VALUES (${userAddress}, ${0}) ON CONFLICT (user_address) DO NOTHING`)
+  } else {
+    query.append(SQL`VALUES (${userAddress}, ${power}) ON CONFLICT (user_address) DO UPDATE SET power = ${power}`)
+  }
+
+  return query
```
[the remark this answers is not in the document]
Comment (kevinszuchet) [reply src/ports/vp/queries.ts:14]: I think it wouldn't be ideal to replace a user's existing VP with a 0 due to a failed query. WDYT?

[the remark this answers is not in the document]
Comment (LautaroPetaccio) [reply src/ports/vp/queries.ts]: Works perfectly!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**